### PR TITLE
Remove callable action badges.

### PR DESCRIPTION
**Pull request by hyoklee**
Callable actions don't work for badges.

### CODE REVIEW

**Review by glennsong09 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -3,8 +3,6 @@ HDF5 version 1.15.0 currently under development
 ![HDF5 Logo](doxygen/img/HDF5.png)
 
 [![develop build status](https://img.shields.io/github/actions/workflow/status/HDFGroup/hdf5/main.yml?branch=develop&label=develop)](https://github.com/HDFGroup/hdf5/actions?query=branch%3Adevelop)
-[![oneAPI CMake build status](https://img.shields.io/github/actions/workflow/status/HDFGroup/hdf5/intel-cmake.yml?branch=develop&label=oneAPI-C)](https://github.com/HDFGroup/hdf5/actions?query=branch%3Adevelop)
-[![oneAPI Autotools build status](https://img.shields.io/github/actions/workflow/status/HDFGroup/hdf5/intel-auto.yml?branch=develop&label=oneAPI-A)](https://github.com/HDFGroup/hdf5/actions?query=branch%3Adevelop)
```
Comment (byrnHDF) [inline README.md:7]: Did you try adding "?event=workflow_call" to the branch and label options?

Like;
intel-cmake.yml?event=workflow_call&branch=develop&label=oneAPI-C

Comment (hyoklee) [reply README.md:7]: @byrnHDF, No. Did you?

Comment (hyoklee) [reply README.md:7]: @byrnHDF, I get the same gray no status icon:

https://img.shields.io/github/actions/workflow/status/HDFGroup/hdf5/intel-cmake.yml?event=workflow_call&branch=develop&label=oneAPI-C

Comment (byrnHDF) [reply README.md:7]: okay, thanks for trying.

**Review by byrnHDF — APPROVED** [review]

**Pull request merged**